app.js

```diff
@@ -4,15 +4,33 @@ var path = require("path");
 var bodyParser = require("body-parser");
 var cookieParser = require("cookie-parser");
 var logger = require("morgan");
+var http = require('http');
 var graphqlHTTP = require("express-graphql");
 var indexRouter = require("./routes/index");
 var usersRouter = require("./routes/users");
-var graphQlSchema = require('./graphql/schema/index')
-var graphQlResolvers = require('./graphql/resolvers/index')
+var graphQlSchema = require("./graphql/schema/index");
+var graphQlResolvers = require("./graphql/resolvers/index");
+var isAuth = require("./middleware/is-auth");
+var Topic = require("./models/topic")
+var User = require('./models/user');
+const { topics } = require("./graphql/resolvers/index");
 require("./config/mongoose");
-
 var app = express();
 
+var server = http.createServer(app);
+var io = require("socket.io")(server);
+
+
+app.use((req, res, next) => {
+  res.setHeader("Access-Control-Allow-Origin", "*");
+  res.setHeader("Access-Control-Allow-Methods", "POST,GET,OPTIONS");
```

Comment:
Other requests are not allowed like DELETE, PATCH ??

Reply:
I was thinking that If I need those requests, I'll add that. Otherwise, it is not required in the current project.